### PR TITLE
CHK-462: Update styling of button in unavailability scenario

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?

Updates the styling of the "remove items from cart" button to be more distinct than the "pickup items" button.

#### What problem is this solving?

These two buttons were a lot similar to each other, which ended up confusing the user of the intent of each button.

#### How should this be manually tested?

[qastore workspace](https://lucas--qastore.myvtex.com/)
[vtexgame1 workspace](https://lucas--vtexgame1.myvtex.com/)

There are two main scenarios which you should test:

> Remember to delete you checkout.vtex.com cookie before entering any of these URLs!

1. [One item unavailable for delivery](https://lucas--qastore.myvtex.com/checkout/cart/add/?sku=454&qty=1&seller=1&sc=1&sku=455&qty=1&seller=1&sc=1&sku=564&qty=1&seller=1&sc=1), use the postal code 22250-040.
2. [One item unavailable for both delivery **and** pickup](https://lucas--qastore.myvtex.com/checkout/cart/add/?sku=454&qty=1&seller=1&sc=1&sku=564&qty=1&seller=1&sc=1&sku=452&qty=1&seller=1&sc=1), also use the postal code 22250-040.

#### Screenshots or example usage

image
<p align="center"><i>Scenario 1</i></p>

image
<p align="center"><i>Scenario 2</i></p>

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "strictBindCallApply": true,
+    "strictFunctionTypes": true,
+    "strictNullChecks": true,
+    "target": "es2017",
+    "types": ["node", "jest"]
```
Comment (wdsrocha) [inline react/tsconfig.json:21]: Why not extend from VTEX tsconfig? https://github.com/vtex/typescript/blob/master/packages/tsconfig/README.md

**Comment by wdsrocha** [comment]
I tried to click to remove the item (second scenario) and nothing was happening. Turns out that the API was returning a 400, but when I refreshed the page, the unavailable item disappeared.

image

**Comment by lucasecdb** [comment]
@wdsrocha I think the issue is because the first request is hanging, and the button isn't disabled while the request is in-flight. We can take a look into this in the future

File: react/tsconfig.json (hunk 1 above)
Comment (lucasecdb) [reply react/tsconfig.json:21]: sure

**Review by wdsrocha — APPROVED** [review]

**Review by jeffersontuc — APPROVED** [review]

**Pull request merged**